webapp/src/containers/WalletPage/__test__/__snapshots__/CreateOrRestoreWalletPage.test.tsx.snap

```diff
@@ -962,8 +962,9 @@ exports[`CreateOrRestoreWalletPage component should check for snapshot 1`] = `
                         </Button>
                         <DownloadCsvModal
                           CsvModalOpen={false}
+                          downloadDisable={true}
                           error=""
-                          filename="Transactions_2021-2-19_14-33-46.csv"
+                          filename="Transactions_2021-2-23_13-51-44.csv"
                           handleCheckBox={[Function]}
                           handleCsvButtonClick={[Function]}
                           handleDownloadWindow={[Function]}
```

Comment:
I feel like snapshot testing is heavily misused in this project. Probably the intention was good, but it has since become a chore of sorts. Useful but not a substitute to E2E.

We should do something about this in Q2.

Reply:
Yep this one is just from an auto update of snapshots